Traceback (most recent call last):
  File "scripts/cloud_experiments.py", line 176, in <module>
    not args.update_function,
AttributeError: 'Namespace' object has no attribute 'update_function'
